.travis.yml

```diff
@@ -1,3 +1,5 @@
 rvm:
-  - 2.1.0
-
+- 2.1.0
+before_script:
+- mkdir ~/.bitpay
```

Comment:
This is actually handled in one of the rake tasks -- hasn't been needed up until now.